content/awell-orchestration/docs/data/data-repository.mdx

```diff
@@ -30,6 +32,93 @@ All the collected data is stored in a single index: `orchestration-datapoint`.
   </p>
 </Alert>
 
+### Sample queries
+
+Given that the data point index is normalised, you can use any combination of the following properties to narrow down the query results:
+- Use `data_set_id` or `pathway.id` to narrow down results to the data collected in a specific pathway
+- Use `patient.id` to narrow down results to data collected for any pathway started for a specific patient
+- Use `data_point_definition.category` to narrow down results to data collected from a specific source category
+- Use `data_point_definition.qualified_key` to narrow down results to a specific data point
```

Comment:
Why is this called qualified_key and not just key like in Awell Studio GUI? I assume they refer to the same: the user-defined identifier for a data point. It's a bit of a pity that one concept has different names in different places.

Reply:
Internally we maintain two properties related to data point keys: the `key` is the resource specific key configured by users and the `qualified_key` is a concatenation of the source key + data point key. Note that even though users are never explicitly told about this qualified key, they do interact with it as the data point selector and data point label always show the qualified key.

When setting up the data point ingestion pipeline I chose to keep both the `key` and `qualified_key` as it allows for different types of queries (`key` can be used to search for a specific data point, `qualified_key` can be used to search by source like all data points in a given form for example).

Changing this would require updating the ingestion pipeline & reimporting data, so that's not really a cheap option.

Reply:
Added an explanation to clarify what qualified keys are.